### ISSUE TITLE
Local mode doesn't find the exec_analysis.sh script

### ISSUE THREAD

**Issue by s-andrews** [DesmondWillowbrook/Librarian#18]
On the latest release there is a bug where the R script is never run because librarian seems to be failing to find the exec_analysis.sh script.

```
librarian/librarian --local example_inputs/RNA.example.fastq
```

Fails with:

```
DEBUG [server::tempdir] Tempdir: "/tmp/tmp.o6ZukmkrFp"
DEBUG [server] Accessing exec_analysis script at path "/bi/home/andrewss/librarian/librarian"
DEBUG [server] Rscript stderr: /bi/home/andrewss/librarian/librarian: /bi/home/andrewss/librarian/librarian: cannot execute binary file

ERROR [server] Rscript failed with status exit status: 126
DEBUG [server] Rscript stdout:
thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: RExit', cli/src/bin/librarian.rs:167:30
```

The error seems to be 

```
Accessing exec_analysis script at path "/bi/home/andrewss/librarian/librarian
```

Because exec_analysis is in /bi/home/andrewss/librarian/scripts

Hopefully it's an easy fix.

**Comment by s-andrews** [comment]
It looks like it's this code at L115 in ```server/lib.rs```

```
let script_path = if cfg!(debug_assertions) {
        PathBuf::from("scripts/exec_analysis.sh")
    } else {
        std::env::current_exe().expect("current executable path should be found")
    };
```

You don't want the current_exe directly - you want the parent folder that the exe is in, and then you need to go down to ```scripts/exec_analysis.sh```

**Comment by DesmondWillowbrook** [comment]
should be fixed by 3ef4c552d31293a227c44abf8e2937203e166ecb
